### PR TITLE
Feature/customisation - camera button

### PR DESCRIPTION
**Pull request by kopijunkie**
## Feature
We want to let integrator specify the background colour of the camera shutter button for hover and active states as the default colours for this button in those states are currently Onfido themed.

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [x] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1363-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1363-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1363-pr-onfido-sdk-ui-onfido.surge.sh

**Review by ethan605 — APPROVED** [review]

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1363-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1363-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1363-pr-onfido-sdk-ui-onfido.surge.sh

File: README.md [hunk 1]
```diff
@@ -544,6 +544,11 @@ A number of options are available to allow you to customize the SDK:
   | `colorBackgroundInfoPill`   | Change background color of Cross Device, Camera/Mic Permissions screens' information header pill |
   | `colorContentInfoPill`      | Change text color of Cross Device, Camera/Mic Permissions screens' information header pill       |
 
+  | Camera Shutter Button               | Description                                                                              |
+  | ----------------------------------- | ---------------------------------------------------------------------------------------- |
+  | `colorBackgroundButtonCameraHover`  | Change background color of Live Selfie/Document Capture screens's Camera button on hover |
+  | `colorBackgroundButtonCameraActive` | Change text color of Live Selfie/Document Capture screen's Camera button on click/tap    |
```
Comment (eraxyz) [inline README.md:550]: Does this change text color or background color?

**Review by eraxyz — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kopijunkie) [reply README.md:550]: Thanks for catching this! Yes, it's background colour that gets changed, not text colour. 🙈

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1363-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1363-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1363-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1363-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1363-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1363-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1363-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1363-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**